IFC4x3/Properties/c/CompressionIndex_1kCf5Yyzb0mg8elIJgHtFu/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CompressionIndex_1kCf5Yyzb0mg8elIJgHtFu" Name="CompressionIndex" UniqueId="6e329162-f3d9-40c2-a228-bd24ea4773f8" PrimaryDataType="IfcReal" />
```

Comment:
ConsistencyIndex is a ratio measure. Why is this one `IfcReal`?

Reply:
IfcReal in Excel spec, but I had the samew question. @j-weil